config/serverless.yml

```diff
@@ -297,3 +297,7 @@ xpack.genAiSettings:
   showAiBreadcrumb: false
   showSpacesIntegration: false
   showAiAssistantsVisibilitySetting: false
+
+# Cross Project Search
+cps.enabled: true
+cps.cpsEnabled: false
```

Comment:
Another approach is to keep the plugin disabled in the main serverless config file and only enable in dev mode. Plugins that need cps enabled can set the cps plugin as an optional dependency until the implementation is complete.

@rudolf WDYT?